dist/index.js

```diff
@@ -38342,13 +38343,15 @@ const run = async () => {
         // Create Issue
         createdIssues.push({ title, body, labels })
       } else {
-        title = `${issueTitlePrefix}${new Date().toTimeString()}`
-        createdIssues[0].title = title
```

Comment:
I think we need to keep this line.

Reply:
No, we don't.

Reply:
My thinking was: in `aggregate` mode, only the first issue is pushed (with `title` set to the first issue's title). But that is not this conditional arm, this is the arm where we add the items' bodies, starting with the second item (if there is a second item, that is). And the conditional I added after this loop will then adjust the `title` (but only if there was more than one item).

So I don't think that we need to set `title` here, as it will be overwritten later, anyway.

Reply:
Another consideration is, of course, whether we should swallow the item titles for aggregated items. Maybe it would be better to do something like this:

```javascript
// do not swallow the items' titles
if (createdIssues.length === 1) createdIssues[0].body = `# ${createdIssues[0].title}\n\n${createdIssues[0].body}`;

createdIssues[0].body += `\n\n# ${title}\n\n${body}`;
```

What do you think?

Reply:
I decided to go for it.

Reply:
>  in `aggregate` mode, only the first issue is pushed […] But that is not this conditional arm

Yes, and I've totally missed that.

Reply:
> Another consideration is, of course, whether we should swallow the item titles for aggregated items. Maybe it would be better to do something like this

Seems like a reasonable idea. I'll take a look at the implementation you pushed.